src/components/TrackFilePicker.demo.js

```diff
@@ -0,0 +1,25 @@
+import Demo, {props as P} from 'react-demo'
+// See https://github.com/rpominov/react-demo for how to make a demo
+
+import React from "react";
+
+import TrackFilePicker from "./TrackFilePicker";
+
+export default (<Demo
+  props={{
+    fileOptions: P.json(["fileA", "fileB", "fileC", "anotherfile.vg"]),
```

Comment:
I think instead of just the filenames, we wanted the input to be a collection of available tracks (so objects including a file and also a track type), right?